pkg/services/app/add.go

```diff
@@ -686,7 +685,7 @@ func (a *App) createPullRequestToRepo(info *AppResourceInfo, repo string, appHas
 		return nil
 	}
 
-	userRepoRef := gitproviders.NewUserRepositoryRef(github.DefaultDomain, owner, repoName)
+	userRepoRef := gitproviders.NewUserRepositoryRef(string(a.GitProvider.GetProviderDomain()), owner, repoName)
```

Comment:
Converting to a string here when it looks like this is already a string?

Reply:
yeah that is something left over from the first draft. Sorry I missed that.